terraform/etl/48-lambda-gov-notify-ingestion.tf

```
@@ -120,3 +128,48 @@ resource "aws_iam_role_policy_attachment" "gov_notify_lambda_secret_access" {
   role       = aws_iam_role.housing_gov_notify_ingestion[0].name
   policy_arn = aws_iam_policy.gov_notify_lambda_secret_access[0].arn
 }
+
+
+module "gov-notify-ingestion-housing-repairs" {
+  count                          = !local.is_production_environment ? 1 : 0
+  source                         = "../modules/aws-lambda"
+  tags                           = module.tags.values
+  lambda_name                    = "govnotify_api_ingestion_repairs"
+  identifier_prefix              = local.short_identifier_prefix
+  handler                        = "main.lambda_handler"
+  lambda_artefact_storage_bucket = module.lambda_artefact_storage_data_source.bucket_id
+  s3_key                         = "govnotify_api_ingestion_repairs.zip"
+  lambda_source_dir              = "../../lambdas/govnotify_api_ingestion_repairs"
+  lambda_output_path             = "../../lambdas/govnotify_api_ingestion_repairs.zip"
+  environment_variables          = {
+    API_SECRET_NAME       = "housing/gov-notify_live_api_key"
+    OUTPUT_S3_FOLDER      = "${module.landing_zone_data_source.bucket_arn}/housing/*"
+    TARGET_S3_BUCKET_NAME = "housing/govnotify/damp_and_mould/"
+  }
+  layers = [
+    "arn:aws:lambda:eu-west-2:120038763019:layer:notifications-python-client-9-0-0-layer:1",
```

this one is the stg account id though so needs to be interpolated to get the right value depending on where you're deploying to 
```suggestion
    "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.data_platform.account_id}:layer:notifications-python-client-9-0-0-layer:1",
```